src/callbacks/log_test_lstm_callback.py

```
@@ -0,0 +1,44 @@
+                    obs,
+                    state=lstm_states,
+                    episode_start=episode_starts,
+                    deterministic=True,
+                )
+                lstm_states = _states
+                obs, rewards, dones, infos = test_env.step(action)
+
+                if dones.any():
+                    info = infos[0]
+                    mlflow.log_metric(
+                        f"eval_test/final_equity",
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)